bigframes/operations/aggregations.py

```diff
@@ -254,6 +254,48 @@ def handles_ties(self):
         return True
 
 
+class QcutOp(WindowOp):
+    def __init__(self, quantiles: typing.Union[int, typing.Sequence[float]]):
+        self.name = f"qcut-{quantiles}"
+        self._quantiles = quantiles
+
+    @numeric_op
+    def _as_ibis(
+        self, column: ibis_types.Column, window=None
+    ) -> ibis_types.IntegerValue:
+        if isinstance(self._quantiles, int):
+            percent_ranks = typing.cast(
+                ibis_types.FloatingColumn,
+                _apply_window_if_present(column.percent_rank(), window),
+            )
+            float_bucket = typing.cast(
+                ibis_types.FloatingColumn, (percent_ranks * self._quantiles)
```

Comment:
There seems to be mypy failures for this and other operations:


bigframes/operations/aggregations.py:272: error: Unsupported operand types for * ("FloatingColumn" and "int")  [operator]
bigframes/operations/aggregations.py:274: error: Argument "lower" to "clip" of "NumericValue" has incompatible type "int"; expected "NumericValue | None"  [arg-type]
bigframes/operations/aggregations.py:274: error: Unsupported operand types for - ("NumericValue" and "int")  [operator]
bigframes/operations/aggregations.py:281: error: Unsupported operand types for < ("FloatingColumn" and "float")  [operator]
bigframes/operations/aggregations.py:284: error: Unsupported operand types for <= ("FloatingColumn" and "float")  [operator]

Reply:
fixed